src/core/split_multimesh_container.gd

```diff
@@ -0,0 +1,47 @@
+class_name SplitMultimeshContainer
```

Comment:
Avoid using class_name in a plugin, otherwise they show up next to every other built-in and user-defined nodes all the time, even when the plugin is disabled.
Preload this script where you need to
`const SplitMultmeshContainer = preload(relative_path)` and use it as usual